### DISCUSSION TITLE
Installing portfoliofy

### DISCUSSION THREAD

**Discussion by faozziyyah** [kaustubhgupta/PortfolioFy#44] [Q&A] [answered]
Hello 👋 
i saw your amazing project (portfoliofy) and I'm so impressed.
I'm trying to install it using the guide in the readme file but i'm not getting it.
please how do I implement / install this project?

thanks so much

**Comment by kaustubhgupta** [comment]
Hi,
Can you explain what issues you are facing?

**Reply by faozziyyah** [reply]
hi, i've been trying to implement something similar to the portfoliofy project based on the guides in the readme file but i'm not getting it, starting from how to create the folder.

**Comment by kaustubhgupta** [comment, marked as answer]
Please check the new documentation of the project, it should now resolve your errors.
